spec/features/welcome_spec.rb

```diff
@@ -29,6 +29,7 @@
 
       expect(page).to have_link("New to Viewing Party? Register Here")
     end
+
     it "And fail to log in is see a flash message" do
```

Comment:
Another minor change: "in is" I think was meant to be "in I"

Reply:
Got it